src/librbd/io/AioCompletion.cc

```diff
@@ -240,9 +239,9 @@ void AioCompletion::complete_request(ssize_t r)
 
 bool AioCompletion::is_complete() {
   tracepoint(librbd, aio_is_complete_enter, this);
-  bool done = (this->state != AIO_STATE_PENDING);
-  tracepoint(librbd, aio_is_complete_exit, done);
-  return done;
+  bool complete = done.load(std::memory_order_acquire);
```

Comment:
It seems like `complete` would be a better name for the class member as well.  I used `done` in my comment only because it's how it was named before commit 77f7c1cf829e5417eb356ebf3bf1fbc8ff7a25a5:
```suggestion
  bool complete = complete.load(std::memory_order_acquire);
```